publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -86,13 +90,42 @@ public UpdatePublicationHandler(ResourceService resourceService,
         this.uriRetriever = uriRetriever;
     }
 
+    private static boolean isAlreadyPublished(Publication existingPublication) {
+        return PublicationStatus.PUBLISHED.equals(existingPublication.getStatus())
+                || PublicationStatus.PUBLISHED_METADATA.equals(existingPublication.getStatus());
+    }
+
+    private static boolean isPending(TicketEntry publishingRequest) {
+        return TicketStatus.PENDING.equals(publishingRequest.getStatus());
+    }
+
+    private static boolean hasMatchingIdentifier(Publication publication, TicketEntry ticketEntry) {
+        return ticketEntry.getResourceIdentifier().equals(publication.getIdentifier());
+    }
+
+    private static URI getCustomerId(Publication publicationUpdate) {
+        return publicationUpdate.getPublisher().getId();
+    }
+
+    private static boolean userIsContributor(RequestInfo requestInfo, UpdatePublicationRequest input) {
```

Comment:
I think the userIsContributor-check should be based on the persisted publication, not the request input